cli/dcos-cassandra/main.go

```diff
@@ -151,6 +151,10 @@ func (cmd *BackupRestoreHandler) runBackup(c *kingpin.ParseContext) error {
 	cli.PrintJSON(cli.HTTPPutJSON("v1/backup/start", string(payload)))
 	return nil
 }
+func (cmd *BackupRestoreHandler) runBackupStop(c *kingpin.ParseContext) error {
```

Comment:
Not opposed to the idea, didn't know enough about the context of these operations to know whether it made sense. I think it'd be straightforward to add. Any preference Mohit?